src/coord/Cargo.toml

```diff
@@ -9,7 +9,7 @@ publish = false
 anyhow = "1.0.32"
 avro = { path = "../avro", package = "mz-avro", features = ["snappy"] }
 aws-util = { path = "../aws-util" }
-backtrace = "0.3.50"
```

Comment:
stray?

Reply:
no, this is intentional, the version was conflicting with whatever version is depended on by pprof, so I made it `0.3` so Cargo would pick the correct thing.

Reply:
Hmm, seems sketchy. The version of backtrace doesn't seem to have changed? Also 0.3.0 is compatible with 0.3.50? Anyway, depending on a recent version of backtrace is intentional; old versions didn't have support for compressed debug info with gimli.

Reply:
I don't understand Cargo version resolution at all so it's 110% possible that I'm just confused, I'll try reverting this line and seeing what happens.

Reply:
ok, now it works, idk what was going on.